cms/collections/jobs.js

```diff
@@ -0,0 +1,389 @@
+    customFields: [],
+  },
+  {
+    id: 'samurai',
+    label: 'Samurai guide',
+    name: 'sam-guide',
+    role: roles.DPS,
+    customFields: [],
+  },
+  {
+    id: 'monk',
+    label: 'Monk guide',
+    name: 'mnk-guide',
+    role: roles.DPS,
+    customFields: [],
+  },
+  {
+    id: 'ninja',
+    label: 'Ninja guide',
+    name: 'nin-guide',
+    role: roles.DPS,
+    customFields: [],
+  },
+  {
+    id: 'dancer',
+    label: 'Dancer guide',
+    name: 'dnc-guide',
+    role: roles.DPS,
+    customFields: [],
+  },
+  {
+    id: 'machinist',
+    label: 'Machinist guide',
+    name: 'mch-guide',
+    role: roles.DPS,
+    customFields: [],
+  },
+  {
+    id: 'bard',
+    label: 'Bard guide',
+    name: 'brd-guide',
+    role: roles.DPS,
+    customFields: [],
+  },
+  {
+    id: 'black-mage',
+    label: 'Black Mage guide',
+    name: 'blm-guide',
+    role: roles.DPS,
+    customFields: [],
+  },
+  {
+    id: 'summoner',
+    label: 'Summoner guide',
+    name: 'smn-guide',
+    role: roles.DPS,
+    customFields: [],
+  },
+  {
+    id: 'red-mage',
+    label: 'Red Mage guide',
+    name: 'rdm-guide',
+    role: roles.DPS,
+    customFields: [],
+  },
+];
+
+const getPathForJob = (job) => `content/jobs/${job.role}/${job.id}`;
+
+const createLandingPage = (job, folderPrefix) => ({
+  label: 'Landing page',
+  name: 'landing',
+  file: `${folderPrefix}${getPathForJob(job)}/_index.md`,
+  fields: [
+    createBodyField(),
+  ],
+});
+
+const createLevelingGuide = (job, folderPrefix) => ({
+  label: 'Leveling Guide',
+  name: 'leveling-guide',
+  file: `${folderPrefix}${getPathForJob(job)}/leveling-guide.md`,
+  fields: [
+    createBodyField(),
+    createThumbnailField(),
+    createAuthorsField(),
+    createPatchField(),
+    createLastUpdatedField(),
+    createChangelogField(),
+  ],
+});
+
+const createBasicGuide = (job, folderPrefix) => ({
+  label: 'Basic guide',
+  name: 'basic-guide',
+  file: `${folderPrefix}${getPathForJob(job)}/basic-guide.md`,
+  fields: [
+    createBodyField(),
+    createThumbnailField(),
+    createAuthorsField(),
+    createPatchField(),
+    createLastUpdatedField(),
+    createChangelogField(),
+  ],
+});
+
+const createSkillsOverview = (job, folderPrefix) => ({
+  label: 'Skills overview',
+  name: 'skills-overview',
+  file: `${folderPrefix}${getPathForJob(job)}/skills-overview.md`,
+  fields: [
+    createBodyField(),
+    createThumbnailField(),
+    createAuthorsField(),
+    createPatchField(),
+    createLastUpdatedField(),
+    createChangelogField(),
+  ],
+});
+
+const createOpeners = (job, folderPrefix) => ({
+  label: 'Openers',
+  name: 'openers',
+  file: `${folderPrefix}${getPathForJob(job)}/openers.md`,
+  fields: [
+    createBodyField(),
+    createThumbnailField(),
+    createAuthorsField(),
+    createPatchField(),
+    createLastUpdatedField(),
+    createChangelogField(),
+  ],
+});
+
+const createFAQ = (job, folderPrefix) => ({
+  label: 'Frequently Asked Questions',
```

Comment:
I'm wondering with the recent discussions of https://github.com/The-Balance-FFXIV/balance-static/issues/589 do we need to add the `layout: 'qna'` into these? My guess is that if we don't then the layout will default to `single.html`?

Just want to future proof any new pages that content creators make

Reply:
Discussed that in Discord. No Changes needed.